### DISCUSSION TITLE
请教博客迁移问题，多谢多谢

### DISCUSSION THREAD

**Discussion by Aaron-boom** [shen-yu/hexo-theme-ayer#120] [Q&A]
由于阿里云的服务器到期，所以将个人网站移到了github.io上，使用hexo+github做的。
我是直接将之前老电脑中_post文件下的markdown博客文件直接拷贝到 新电脑的_post文件夹下，再使用hexo g --d,导致新的博客中的照片无显示。
图片我是使用域名+七牛云做了加速配置。
请问下大神这个怎么解决？

**Comment by Aaron-boom** [comment]
您好！
![image](https://user-images.githubusercontent.com/55744718/116976937-60be1600-acf4-11eb-9222-4cd5da8f9104.png)

我新建一篇文章测试了下，生成的html文件中的照片是这样的。

**Comment by shen-yu** [comment]
应该是图片使用了相对路径，如果用的是cdn绝对路径是可以显示的

**Reply by Aaron-boom** [reply]
多谢博主的解答啦~我后来思考了下，因为github部署后生成的url使用https协议，而我的七牛云的图片超链是http协议的，谷歌浏览器对于http的支持感觉不太友好。我的解决是将七牛云的超链升级为https协议了，解决了这个bug。不过我还是不清楚具体的原因（有人说是域名污染），这个我自己还在研究。

**Reply by shen-yu** [reply]
是的，chrome对http支持并不友好，主要是因为安全性，建议使用https
